allen-cell-animated/cell-feature-explorer
# Fix/move small screen warning off of landing page

Problem
=======
Partially addresses: [mobile: don't load data until "OK" on warning screen](https://aicsjira.corp.alleninstitute.org/browse/ANIMCELLGR-625)

I'm trying to make smaller PRs. The first thing we need to fix is the fact that the user gets a small screen warning on the CFE landing page. We don't need the warning to appear until they're trying to access the actual CFE viewer.

Solution
========
I moved all the code pertaining to the small screen warning from the `App` container to `Cfe`. This is all just cut and paste.

## Type of change
Please delete options that are not relevant.


Steps to Verify:
----------------
1. Pull this branch and `npm start`
2. Make your window super narrow on the landing page. No warning should pop up.
3. Click on either of the library cards to go to the actual CFE. Warning should pop up.
